zone/client_packet.cpp

```
@@ -5811,11 +5811,10 @@ void Client::Handle_OP_Disarm(const EQApplicationPacket *app) {
 
 	Disarm_Struct *disarm = (Disarm_Struct *)app->pBuffer;
 
-	// RoF2+ Disarm shares the secondary combat-ability lane with Tiger Claw.
+	// Disarm shares the secondary combat-ability lane with Punch/Bash abilities.
 	if (
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)